front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/AuthorizationSupport.groovy

```diff
@@ -31,15 +32,34 @@ class AuthorizationSupport {
 
   boolean hasRunAsUserPermission(Pipeline pipeline) {
     List<String> runAsUsers = pipeline.triggers*.runAsUser
```

Comment:
`pipeline.triggers.findResults { it.runAsUser }` should avoid the need to `.removeAll([null])`